express/code/blocks/prompt-marquee/prompt-marquee.css

```diff
@@ -0,0 +1,351 @@
+}
+
+.prompt-marquee .column-picture img {
+  width: 100%;
+  max-width: 100%;
+  aspect-ratio: var(--prompt-marquee-media-aspect-ratio);
+  object-fit: cover;
+}
+
+@media (min-width: 768px) {
+  .prompt-marquee .column-picture {
+    margin-top:  0px;
+  }
+}
+
+
+@media (min-width: 768px) {
+  .prompt-marquee {
+    max-width: unset;
+    margin: 0;
+    padding-left: var(--spacing-500);
+    padding-right: var(--spacing-500);
+  }
+
+  .prompt-marquee > div {
+    gap: var(--spacing-700);
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .prompt-marquee > div > div {
+    width: 50%;
+    box-sizing: border-box;
+  }
+
+  .prompt-marquee .column-picture:not(.column-picture-mobile) {
+    display: block;
+  }
+
+  .prompt-marquee .column-picture-mobile {
+    display: none;
+  }
+
+  /* Paragraphs inherit mobile base sizing; override here if needed */
+
+  .prompt-marquee .cta-with-input { flex-direction: row; align-items: center; }
+  .prompt-marquee .cta-with-input .prompt-marquee-input { width: auto; flex: 1 1 auto; }
+  .prompt-marquee .cta-with-input .button,
+  .prompt-marquee .cta-with-input .con-button { width: auto; order: 1; min-width: auto; }
+
+  .prompt-marquee .column-picture::before {
+    width: 160%;
+    height: 160%;
+    right: -30%;
+    top: -30%;
+    left: auto;
+  }
+}
+
+/* Keep one-row layout on lower tablet and shrink input text to avoid truncation */
+@media (min-width: 768px) and (max-width: 900px) {
+  .prompt-marquee .cta-with-input { flex-wrap: nowrap; }
+  .prompt-marquee .cta-with-input .prompt-marquee-input { 
+    font-size: var(--body-font-size-s);
+    min-width: 0; /* allow flex item to shrink */
+  }
+  .prompt-marquee .cta-with-input .button,
+  .prompt-marquee .cta-with-input .con-button {
+    width: auto;
+    white-space: nowrap;
+    min-width: auto; /* keep compact; rely on padding only */
+  }
+}
+
+@media (min-width: 900px) {
+  .prompt-marquee {
+    padding-top: var(--prompt-marquee-desktop-padding-block);
+    padding-bottom: var(--prompt-marquee-desktop-padding-block);
+  }
+
+  .prompt-marquee > div > .column-picture:not(.column-picture-mobile) {
+    width: auto;
+    flex: 0 1 var(--prompt-marquee-desktop-media-max-width);
+    max-width: var(--prompt-marquee-desktop-media-max-width);
+  }
+}
+
+@media (min-width: 1280px) {
+  .prompt-marquee {
+    margin: auto;
+    max-width: 1400px;
+    padding-left: var(--spacing-600);
+    padding-right: var(--spacing-600);
+  }
+  .prompt-marquee > div {
+    padding-left: var(--spacing-600);
+    padding-right: var(--spacing-600);
+  }
+
+  .prompt-marquee h1 {
+    font-size: var(--heading-font-size-xl);
+    line-height: var(--heading-line-height);
+  }
+
+  .prompt-marquee .column p {
+    font-size: var(--body-font-size-l); 
+  }
+}
+
+/* Input styling to match Milo form look */
+.prompt-marquee-input {
+  height: 48px;
+  box-sizing: border-box;
+  border-radius: 10px;
+  border: 2px solid var(--Palette-gray-300, #DADADA);
```

Comment:
Seeing this pattern a few times in this PR as well
Should be `var(--Palette-gray-300)` not hard coded values